croud/session.py

```diff
@@ -79,6 +87,31 @@ def __init__(
 
         return await resp.json()
 
+    async def rest_fetch(
+        self, method: RequestMethod, params: Optional[JsonDict], endpoint: str
+    ) -> JsonDict:
+        url = self.url + CLOUD_APP_API_PREFIX + endpoint
+
+        resp = await getattr(self.client, method.value)(
+            url, params=params, allow_redirects=False
+        )
+
+        if resp.status == 302:  # login redirect
+            print_error("Unauthorized. Use `croud login` to login to CrateDB Cloud.")
+            await self.client.close()
+            exit(1)
+
+        if resp.status < 200 or resp.status > 299:
```

Comment:
I don't mind either way. This might be easier to grasp.
```suggestion
        if not (200 <= resp.status <= 299):
```